web/themes/custom/novel/novel.theme

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * @file
+ * Novel Theme.
+ */
+
+/**
+ * Implements template_preprocess_page().
+ *
+ * @param mixed[] $variables
+ *   Theme variables.
+ */
+function novel_preprocess_page(array &$variables): void {
+  $data = [
+    'search-header-url' => '/search',
+    'alt-text' => t('Search field'),
+    'input-placeholder' => t('Start typing in order to search'),
```

Comment:
Please add a context. This is required [according to our coding guidelines](https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/main/documentation/code-guidelines.md#translations).

We should consider requesting some form of automation alerting developers of this. I think many people will forget this.

Reply:
👍